docs/man/docker-inspect.1.md

```diff
@@ -30,69 +30,109 @@ each result.
 
 To get information on a container use it's ID or instance name:
```

Comment:
`it's` > `its` can you change that while  you are in there.  Also, why was it necessary to change this example output so extensively?